src/prop.ts

```diff
@@ -23,6 +23,7 @@ export interface BasePropOptions {
 
 export interface PropOptions extends BasePropOptions {
   ref?: any;
+  refType?: any;
```

Comment:
Is ˋanyˋ really correct here? What about ˋNumber | String | Bufferˋ?

Reply:
No, you're right. It uses a mongoose.Schema.Types object, but I'll find the correct way to type it.